src/CoreApi/PubSubApi.cs

```diff
@@ -48,7 +48,7 @@ public Task PublishAsync(string topic, byte[] message, CancellationToken cancel
             url.Append("/api/v0/pubsub/pub");
             url.Append("?arg=");
             url.Append(System.Net.WebUtility.UrlEncode(topic));
-            url.Append("&arg=");
+            url.Append("&data=");

```

Comment:
This commit does not conform to the 0.0.11 API changes and should be disregarded entirely.